Sources/Datadog/DatadogCore/DatadogCore.swift

```diff
@@ -193,7 +193,7 @@ internal final class DatadogCore {
     /// A list of storage units of currently registered Features.
     private var allStorages: [FeatureStorage] {
         let v1Storages = v1Features.values.compactMap { $0 as? V1Feature }.map { $0.storage }
```

Comment:
question/ Do we still have V1 Features 🤔💭?

Reply:
Yes, currently Tracing and RUM are still v1, but not for long ;)